packages/circuit-ui/package.json

```diff
@@ -49,6 +49,7 @@
     "@emotion/styled": "^11.10.6",
     "@sumup/collector": "^1.6.1",
     "@sumup/design-tokens": "^5.0.0",
+    "@sumup/foundry": "^6.1.0",
```

Comment:
Foundry is already installed and configured at the workspace root. Please revert all Foundry-related changes.